recovery-funds.Rmd

```diff
@@ -50,20 +50,25 @@ librarian::shelf(
   zoo,
   palettes,
   xaringanExtra,
-  dotenv
+  dotenv,
+  sknifedatar
 )
 
-# remotes::install_version("sknifedatar", version = "0.1.2", repos = "http://cran.us.r-project.org")
-library(sknifedatar)
+# Set the s3_bucket environment variable
+Sys.setenv(s3_bucket = "federal-funds-equity")
```

Comment:
We generally try to avoid including bucket names in public code (a bit fuzzy here since the bucket is public, but still the suggestion below is best practice. Note this relies on you having the .env file saved in the root of the repo. 

```suggestion
load_dot_env(file = ".env")
```

Reply:
Thanks. I've removed it from open code.